pyproject.toml

```diff
@@ -45,6 +45,7 @@ qrcode = {version = "^7.3.1", extras = ["pil"]}
 
 [tool.poetry.extras]
 docs = ["recommonmark", "sphinx"]
+production = ["psycopg2"]
```

Comment:
This isn't only required for production. Our CI tests also use postgres as database. So I really feel like this is not the right name. "Postgres" might be more suitable?